projects/tinkerbell/tink/docker/linux/tink-relay-init/Dockerfile

```diff
@@ -0,0 +1,18 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-nsenter
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as builder
+
```

Comment:
You probably need to add a `WORKDIR /newroot` instruction here, so that the installation happens there, else the `COPY` instruction on line 19 might fail.

Reply:
done